glide.lock

```diff
@@ -1,27 +1,50 @@
-hash: 22996bdfd992015c6b999771d64996343958df212db5c82b032b8142caaa30ff
-updated: 2017-01-07T22:32:44.56183669-05:00
+hash: a09d6c23e8f9b633b9665c587959bea7db86ddb4afac6456ba4bf7dee4f8d57e
+updated: 2017-04-12T17:09:38.560840683-04:00
 imports:
 - name: github.com/facebookgo/clock
   version: 600d898af40aa09a7a93ecb9265d87b0504b6f03
+- name: github.com/golang/mock
+  version: bd3c8e81be01eef76d4b503f5e687d2d1354d2d9
+  subpackages:
+  - gomock
+  - mockgen
 - name: github.com/golang/protobuf
-  version: 3852dcfda249c2097355a6aabb199a28d97b30df
+  version: 7390af9dcd3c33042ebaf2474a1724a83cf1a7e6
   subpackages:
   - proto
+- name: github.com/m3db/m3cluster
+  version: 9cd63fa5a295147b9cddd3295f37f8400bcae415
+  subpackages:
+  - kv
 - name: github.com/m3db/m3x
-  version: d39b3685ef8430f7fbeedc666f7da781d7f55ba9
+  version: d19f7459d421a726af8ecd086015be7e4f327de5
   subpackages:
+  - checked
+  - close
   - instrument
   - log
   - pool
   - time
+  - watch
 - name: github.com/uber-go/tally
-  version: 1f0f7171f312a8cff5f697fe5f4d45dbaec8c3ac
+  version: 08e86c3c1eb55364d563bd6beb982040225f4bb7
+- name: go.uber.org/atomic
+  version: 3b8db5e93c4c02efbc313e17b2e796b0914a01fb
+- name: go.uber.org/zap
```

Comment:
I was originally going to use `zap` then Chao pointed out that we have our own logging package so I switched to that (`xlog`). I forgot to update Glide so I'll remove these dependencies since they're unnecessary now.